stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -144,9 +169,16 @@ public static String asFundingType(@Nullable String possibleFundingType) {
     @Nullable
     @TokenType
     public static String asTokenType(@Nullable String possibleTokenType) {
+        if (isBlank(possibleTokenType)) {
```

Comment:
Yes -- I was thinking of this like some of the other `@StringDef` types where there is a `.UNKNOWN` value, and I wanted `null` to be different from that.